File: app/src/main/java/com/androidwind/androidquick/sample/JavaTestActivity.java
package com.androidwind.androidquick.sample;

import android.os.Bundle;

import com.androidwind.androidquick.util.manager.QActivity;
import com.androidwind.androidquick.ui.receiver.NetStateReceiver;
import com.androidwind.androidquick.util.LogUtil;
import com.androidwind.androidquick.util.ToastUtil;

import org.jetbrains.annotations.NotNull;
import org.jetbrains.annotations.Nullable;

/**
 * @author ddnosh
 * @website http://blog.csdn.net/ddnosh
 */
public class JavaTestActivity extends BaseActivity {
    @Override
    protected int getContentViewLayoutID() {
        return 0;
    }

    @Override
    protected void getBundleExtras(@NotNull Bundle extras) {

    }

    @Override
    protected void initViewsAndEvents(@Nullable Bundle savedInstanceState) {
        super.initViewsAndEvents(savedInstanceState);
        getBinding();
        LogUtil.i(TAG, "");
        ToastUtil.showToast("");
        QActivity.addActivity(this);
        NetStateReceiver.unRegisterNetworkStateReceiver(this);
        showLoadingDialog();
    }
}

File: app/src/main/java/com/github/ddnosh/arabbit/sample/other/JavaTestActivity.kt
package com.github.ddnosh.arabbit.sample.other

import android.os.Bundle
import com.github.ddnosh.arabbit.sample.R
import com.github.ddnosh.arabbit.sample.base.BaseActivity
import com.github.ddnosh.arabbit.util.LogUtil
import com.github.ddnosh.arabbit.util.ToastUtil.showToast
import com.github.ddnosh.arabbit.util.manager.QActivity.addActivity

/**
 * @author ddnosh
 * @website http://blog.csdn.net/ddnosh
 */
class JavaTestActivity : BaseActivity() {
    override val contentViewLayoutID: Int = R.layout.activity_test

    override fun initViewsAndEvents(savedInstanceState: Bundle?) {
        LogUtil.i(TAG, "")
        showToast("")
        addActivity(this)
        showLoadingDialog()
    }
}